src/bundle/Resources/public/js/scripts/fieldType/ezdate.js

```diff
@@ -73,10 +73,13 @@
             return;
         }
 
+        const selectedDateWithUserTimezone = eZ.helpers.timezone.convertDateToTimezone(date[0]);
+
         date = new Date(date[0]);
         date = new Date(Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()));
+        const microTime = date.getTime() - (selectedDateWithUserTimezone.utcOffset() * 60 * 1000);
```

Comment:
Operator precedence makes it unecessary.
```suggestion
        const microTime = date.getTime() - selectedDateWithUserTimezone.utcOffset() * 60 * 1000;
```